reana_workflow_controller/workflow_run_manager.py

```diff
@@ -623,12 +624,21 @@ def _create_job_spec(
             self.workflow.workspace_path
         )
 
+        print(f"creating workflow. using kueue: {USE_KUEUE}")
+
+        labels = {
```

Comment:
BTW please add Xavier Tintin (@xaviertintin) to the AUTHORS.md file with the ORCID https://orcid.org/0000-0002-3150-9112 You can also amend the commit to add `Co-authored-by: Xavier Tintin <alextintin007@outlook.com>` and close his former pull request #585.